src/lib/ParseApp.js

```diff
@@ -357,8 +357,13 @@ export default class ParseApp {
   }
 
   clearCollection(className) {
-    let path = `/apps/${this.slug}/collections/${className}/clear`;
-    return AJAX.del(path);
+    if (this.serverInfo.parseServerVersion == 'Parse.com') {
+      let path = `/apps/${this.slug}/collections/${className}/clear`;
```

Comment:
We don't actually need to keep the version that goes to Parse.com around, it doesn't work anyway.